### PR TITLE
Enhance/lu 2454 es enrichment

### PR DESCRIPTION
**Pull request by kingmushroom**
All review points *hopefully* addressed.

### CODE REVIEW

File: tests/test_enrichment.py [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+        queue_url = sqs.get_queue_by_name(QueueName="test_queue").url
+        # Method
+        with mock.patch.dict(
+            lambda_wrangler_function.os.environ,
+            {"checkpoint": "1", "queue_url": queue_url},
+        ):
+            lambda_wrangler_function.lambda_handler(
+                {"RuntimeVariables": {"checkpoint": 666}}, None
+            )
+            self.assertRaises(ValueError)
```
Comment (bitmonkey) [inline tests/test_enrichment.py:344]: Should probably be a bit more specific here and check for what you're expecting to see failed....i.e the message

**Review by bitmonkey — COMMENTED** [review]
Had to fix a test class name to be correctly cased and if am being niggly the marshmallow test should really check for the things it expected to fail i.e the failure message should be checked.

One thing that looks a little odd is I think the serverless.yml you've checked in shouldn't be there?

File: serverless.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+service: imputation
```
Comment (bitmonkey) [inline serverless.yml]: Don't think this file should be here - seems to be defining step functions for movement and means and nothing to do with enrichment.

**Comment by kingmushroom** [comment]
At 647am on a sunday. Now that is commitment.

**Comment by bitmonkey** [comment]
@kingmushroom Looking good - `flake8` still complaining about line length's though, will come and show you how to work around this....

**Comment by bitmonkey** [comment]
@kingmushroom Happy days....merging now :tada:

**Review by bitmonkey — APPROVED** [review]

**Pull request merged**